pii_secret_check_hooks/check_file/base_content_check.py

```diff
@@ -39,8 +39,9 @@ def __init__(
         self,
         check_name,
         allow_changed_lines=False,
-        excluded_file_list=[],
+        excluded_file_list=None,
```

Comment:
Good catch! Could we do the assignment on the self var and save some lines of code?